db/migrations/versions/d7eed6243143_.py

```diff
@@ -1,151 +0,0 @@
-"""empty message
-
-Revision ID: d7eed6243143
-Revises:
-Create Date: 2022-09-28 16:47:01.367701
-
-"""
-import sqlalchemy as sa
-import sqlalchemy_utils
-from alembic import op
-
-
-# revision identifiers, used by Alembic.
-revision = "d7eed6243143"
-down_revision = None
-branch_labels = None
-depends_on = None
-
-
-def upgrade():
-    # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table(
```

Comment:
If we are deleting migrations, I guess we'll need to re-create the database on test?

Reply:
Yes, because the whole schema has changed since we last merged to main.